public/locales/pt/common.json

```
@@ -1,64 +1,64 @@
 {
-  "scrobble": "Scrobble",
-  "getScrobbling": "Iniciar o scrobbling",
-  "loading": "Loading...",
+  "scrobble": "Fazer scrobble",
+  "getScrobbling": "Comece a fazer scrobble",
+  "loading": "Carregando...",
   "artist": "Artista",
-  "song": "Som",
+  "song": "Música",
```

Would `Música` refer to a song (i.e. a single track) better than `Som`? Some googling suggests using `Faixa` instead, but I'm no native speaker. As far as I understand, and if the meaning is similar to the Spanish one, `Música` refers to music in general, not to a single track.

---

Yes. If the original was `Track` then it would directly translate to `Faixa`. `Som` is a more broader term, could mean any kind of `sound`.

---

```suggestion
  "song": "Faixa",
```

ok! Can we got with `Faixa` then?

---

I'll be fixing it in a future commit :+1: